PicoProducer/python/batch/submit_HTCondor.sub

```diff
@@ -23,7 +23,9 @@ getenv                = CMSSW*, SCRAM_ARCH*, *CONTAINER
 environment           = "JOBID=$(ClusterId) TASKID=$(ProcId) CONTAINER=$(container)"
 # NOTE: The following OS requirement should be replaced for CMSSW 13 & later,
 #       or replaced with singularities all together when CentOS7 is phased out on lxplus
-requirements          = (OpSysAndVer =?= "CentOS7")
+#       See https://batchdocs.web.cern.ch/local/submit.html#os-choice
+#requirements          = (OpSysAndVer =?= "CentOS7")
+#requirements          = (OpSysAndVer =?= "AlmaLinux9")
```

Comment:
Should the requirement of AlmaLinux9 be uncommented by default?

Reply:
Hi @mbluj, at the moment AlmaLinux9 seems to be the the default node: https://batchdocs.web.cern.ch/local/submit.html#os-choice
so I think it is better to leave it like this so it can be reused by other computing systems with HTCondor.

If AlmaLinux9 is inevitably phased out in the future, we'll have to rethink this again, or make it easier to use singularities or whatever.

I am working on redesigning the TauFW configuration, and in the medium term I will make it easier for the user to configure this line from YAML files, instead of changing this hardcoded lines `.sub` file.